projects/packages/jetpack-mu-wpcom/src/features/wpcom-plugins/wpcom-plugins.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Adds a tiny WordPress.com Plugins integration to the plugin list.
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+/**
+ * Displays a banner before the plugin browser that links to the WP.com Plugins Marketplace.
+ */
+function wpcom_plugins_show_banner() {
+
+	// phpcs:ignore WordPress.Security.NonceVerification.Recommended
+	if ( isset( $_GET['tab'] ) && 'favorites' === $_GET['tab'] ) {
+		// no banner on the favorites tab, it's a bit overbearing given they presumably want
+		// something specific.
+		return;
+	}
+
+	// No banner for agency-managed sites.
+	if ( ! empty( get_option( 'is_fully_managed_agency_site' ) ) ) {
+		return;
+	}
```

Comment:
This check is no longer required, as we already gate it inside `load_wpcom_user_features()` 😄